crawler/sql_queries.py

```diff
@@ -80,7 +87,8 @@
 lh_sample_uuid=VALUES(lh_sample_uuid),
 lh_source_plate_uuid=VALUES(lh_source_plate_uuid),
 must_sequence=VALUES(must_sequence),
-preferentially_sequence=VALUES(preferentially_sequence);
+preferentially_sequence=VALUES(preferentially_sequence),
+is_current=VALUES(is_current);
```

Comment:
The last of the new unit tests fails without this change, proving it's the source of our issue.